core/src/main/java/org/sonarsource/sonarlint/core/embedded/server/ShowIssueRequestHandler.java

```diff
@@ -185,23 +187,28 @@ static ShowIssueQuery extractQuery(ClassicHttpRequest request) {
     } catch (URISyntaxException e) {
       // Ignored
     }
-    return new ShowIssueQuery(params.get("server"), params.get("project"), params.get("issue"));
+    return new ShowIssueQuery(params.get("server"), params.get("project"), params.get("issue"), params.get("branch"), params.get("pullRequest"));
   }
 
   @VisibleForTesting
   public static class ShowIssueQuery {
     private final String serverUrl;
     private final String projectKey;
     private final String issueKey;
+    private final String branch;
```

Comment:
For clarification, do we always expect to have the branch parameter? I thought in case of master/main branch it would be empty as of today

Reply:
Yes, 'branch' will always be there, even for the default branch.